core/Piranha/Repositories/IMediaRepository.cs

```diff
@@ -32,13 +40,31 @@ public interface IMediaRepository
         /// <returns>The available media folders</returns>
         Task<IEnumerable<Guid>> GetAllFolders(Guid? folderId = null);
 
+        /// <summary>
+        /// Get media for all Ids in this enumerable.
+        /// </summary>
+        /// <param name="mediaIds"></param>
+        /// <returns></returns>
+        Task<IEnumerable<Media>> GetAllByIdAsync(IEnumerable<Guid> mediaIds);
```

Comment:
Without analyzing where these methods are called, do we need two methods in the repository with the same params (`IEnumerable` / `params Guid[]`)?

Reply:
You make a good point, especially since in either case I usually transform into an array first anyway to ensure that the queryable can actually translate it into SQL. Will be adjusting it.